BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -637,7 +653,38 @@ extension NSLineBreakMode {
     }
 }
 
+
 extension NSAttributedString {
+
+    fileprivate static let invalidMultiLineModes: Set<NSLineBreakMode> = [.byTruncatingHead, .byTruncatingMiddle]
+    fileprivate static let invalidSingleLineModes: Set<NSLineBreakMode> = [.byCharWrapping, .byWordWrapping]
+
+    @_spi(BlueprintAttributedLabel)
+    /// Call this method to set `needsTextNormalization`, as an optimization in your custom label implementation
```

Comment:
`normalizingForView` is a bit under-explained. Since this method is now public (if qualified), we may also wish to explain that normalization 

- applies a consistent font to the portions of the attributed label which have no font
- ensures full control of link styling by providing a custom link attribute
- updates line break mode in cases where the provided value is nonsensical

Reply:
This first bit makes me a bit worried about opting out of normalization, and is probably the main caveat to consumer usage?

Reply:
True – I can add more docs here – mostly left it as is since in practice, we are the only ones touching this or caring about it.